src/config/color.py

```diff
@@ -61,9 +80,7 @@ def __init__(self):
             val_dataloader,
             test_dataloader,
             total_epochs,
-            batch_size,
-            point_batches,
-            logger,
+            metric_logger,
             storage
         )
 
```

Comment:
Is this file a part of this pull request? If it's a remnant of other colorization pr maybe it makes sense to have that branch as a target of this pr?